Gemfile

```diff
@@ -13,7 +13,7 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
 end
 
 gem 'coderay' # for syntax highlighting
-gem 'yard', '~> 0.9.12', :require => false
+gem 'yard', '~> 0.9.22', :require => false
```

Comment:
Can we limit this to 2.7.0 only until we resolve the docs differences? Unless you want to go through the diff now?